chrome/content/zotero/integration/quickFormat.js

```diff
@@ -79,15 +88,46 @@ var Zotero_QuickFormat = new function () {
 			
 			new WindowDraggingElement(document.querySelector("window.citation-dialog"), window);
 			
-			qfs = document.querySelector(".citation-dialog.search");
-			qfi = document.querySelector(".citation-dialog.iframe");
 			qfb = document.querySelector(".citation-dialog.entry");
+			qfe = document.querySelector(".citation-dialog.editor");
```

Comment:
Oh, and also I renamed `panel`, `panelLocator`, `panelLocatorLabel`, etc. for `citationPopup`, `citationPopupLocator`, etc. To avoid confusion between `panel` (as in the citation details popup) and `referencePanel` (as in the references list) that I myself fell for earlier

Reply:
> Oh, and also I renamed `panel`, `panelLocator`, `panelLocatorLabel`, etc. for `citationPopup`, `citationPopupLocator`, etc. To avoid confusion between `panel` (as in the citation details popup) and `referencePanel` (as in the references list) that I myself fell for earlier

You will have to rename those again, as those are item popups, not citation popups. I also prefer popover over popup, as it's more specific (popup is a generic term that can mean anything from an icon popover, to a full blown new window dialog).

Reply:
That makes sense, `itemPopover` it is!